frontend/src/tests/setup.ts

```
@@ -1,151 +1,151 @@
-		return {
-			width,
-			height,
-			data: new Uint8ClampedArray(width * height * 4),
-		};
-	}
+  constructor(width: number, height: number) {
+    return {
+      width,
+      height,
+      data: new Uint8ClampedArray(width * height * 4),
+    };
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)